src/miniclient/client.go

```diff
@@ -81,7 +85,8 @@ func (mm *Conn) Pipe(pipe string) (io.Reader, io.WriteCloser) {
 		PlumbPipe: pipe,
 	})
 	if err != nil {
-		log.Fatal("local pipe gob encode: %v", err)
+		mm.err = fmt.Errorf("local pipe gob encode: %v", err)
```

Comment:
Pipe is a bit weird -- it's only invoked when calling `minimega -pipe`. I left it unchanged but we may want to tweak it in the same way later.